### PR TITLE
Adding doc comments to the chain contracts

### PR DESCRIPTION
**Pull request by vladbochok**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol [hunk 1]
```diff
@@ -36,7 +36,7 @@ contract MailboxFacet is ZkSyncHyperchainBase, IMailbox {
     string public constant override getName = "MailboxFacet";
 
     /// @dev Era's chainID
-    uint256 public immutable ERA_CHAIN_ID;
+    uint256 internal immutable ERA_CHAIN_ID;
```
Comment (StanislavBreadless) [inline l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol:39]: are you sure? it is not too big of a deal, but it is just easier to double check the correctness of values when they are public

File: l2-contracts/contracts/bridge/L2SharedBridge.sol [hunk 2]
```diff
@@ -39,10 +39,10 @@ contract L2SharedBridge is IL2SharedBridge, Initializable {
     /// This is non-zero only on Era, and should not be renamed for backward compatibility with the SDKs.
     address public override l1Bridge;
 
-    uint256 public immutable ERA_CHAIN_ID;
-
     /// @dev Contract is expected to be used as proxy implementation.
     /// @dev Disable the initialization to prevent Parity hack.
+    uint256 internal immutable ERA_CHAIN_ID;
```
Comment (StanislavBreadless) [inline l2-contracts/contracts/bridge/L2SharedBridge.sol:44]: same here, I would keep it public

File: l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol (hunk 1 above)
Comment (vladbochok) [reply l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol:39]: I am 99% sure, as it is a diamond facet, should not get a public view method, no?

File: l2-contracts/contracts/bridge/L2SharedBridge.sol (hunk 2 above)
Comment (vladbochok) [reply l2-contracts/contracts/bridge/L2SharedBridge.sol:44]: Here it can be public, agree

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**